RecoCTPPS/TotemRPLocal/plugins/CTPPSLocalTrackLiteProducer.cc

```diff
@@ -101,7 +102,7 @@ CTPPSLocalTrackLiteProducer::produce( edm::Event& iEvent, const edm::EventSetup&
       for ( const auto& trk : rpv ) {
         if ( !trk.isValid() ) continue;
         pOut->emplace_back( rpId, trk.getX0(), trk.getX0Sigma(), trk.getY0(), trk.getY0Sigma(), trk.getTx(), trk.getTxSigma(), trk.getTy(), trk.getTySigma(), 
-        trk.getChiSquaredOverNDF(), ReconstructionInfo::invalid, trk.getNumberOfPointsUsedForFit(),0,0 );
+        trk.getChiSquaredOverNDF(), CTPPSpixelLocalTrackReconstructionInfo::invalid, trk.getNumberOfPointsUsedForFit(),0,0 );
```

Comment:
I see that rounding was not applied to the strip tracks.
Was this intentional?

Reply:
No, it will be added, too.

Reply:
thanks for the heads up on this.
I'll wait for the update then.

Reply:
> No, it will be added, too.

Please let me know when this update is expected.